eth_tester/utils/backend_testing.py

```diff
@@ -1208,6 +1208,31 @@ def test_delete_filter(self, eth_tester):
         with pytest.raises(FilterNotFound):
             eth_tester.delete_filter(12345)
 
+    #
+    # Serializer
+    #
+    def test_receipt_gas_used_computation(self, eth_tester):
+        eth_tester.disable_auto_mine_transactions()
+
+        tx_hashes = []
+        for i in range(4):
+            tx = {
+                'from': eth_tester.get_accounts()[i],
+                'to': eth_tester.get_accounts()[i + 1],
+                'gas': (i + 1) * 20000 + 10000,
```

Comment:
The fact that this is calculated makes it seem like the gas amount is relevant to the test, but it isn't AFAICT. This would be just as effective as a constant, like `30000`, right?